src/main/scala/com/typesafe/sbt/bundle/SbtBundle.scala

```diff
@@ -688,7 +688,12 @@ object SbtBundle extends AutoPlugin {
     def toString[T](valueAndConfigName: (T, Option[String]), f: T => String): (String, Option[String]) =
       f(valueAndConfigName._1) -> valueAndConfigName._2
 
-    val componentPrefix = s"""components."${(normalizedName in config).value}""""
+    def escapeStartCommand(startCommand: Seq[String]): String = {
+      val executable = startCommand.head
+      val arguments = startCommand.tail
+      val executableEscaped = executable.replace("\\", "/")
+      formatSeq(Seq(executableEscaped) ++ arguments)
+    }
```

Comment:
Perhaps it'd be better to escape the path where we build it do that we don't have to head and tell. We could introduce a new setting maybe (normalizedStartCommand).